ui/src/plugins/dev.perfetto.ExplorePage/styles.scss

```diff
@@ -133,3 +133,72 @@
     margin-bottom: 16px;
   }
 }
+
+.query-canvas-container {
+  position: relative;
+  height: 800px;
+  background-color: lightgray;
+  overflow: auto;
+  border: 1px solid gray;
+  border-radius: 4px;
+}
+
+.node-box {
+  border: 2px solid blue;
+  border-radius: 5px;
+  padding: 10px;
+  cursor: grab;
+  background-color: lightblue;
+  position: absolute;
+  display: flex;
+  align-items: center;
+  gap: 4px;
+
+  &.selected {
+    border-color: var(--perfetto-selection-color);
+    border-color: yellow;
+    box-shadow: 0 0 5px yellow;
+  }
+
+  &.invalid {
+    border-color: red;
+    background-color: lightcoral;
+  }
+
+  // Style for the popup menu button inside the node box
+  .pf-button {
+    margin-left: auto;
+  }
+}
+
+.query-canvas-add-button-container {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+
+  .pf-button {
+    height: 100px;
+    width: 100px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 48px;
+  }
+}
+
+.right-panel {
```

Comment:
This is a very simple selector, could apply to places you don't necessarily want it to apply to.

To follow BEM, it should be something like `.pf-explore-page__right-panel`. I'm not going to die on this hill, but I would prefer it.

See https://getbem.com/